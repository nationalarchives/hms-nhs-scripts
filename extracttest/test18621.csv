data.text

has Ship
HaS Ship
Has Ship
has Ship
haS Ship
i has Ship
i HaS Ship
i Has Ship
i has Ship
i haS Ship
Ship has
Ship HaS
Ship Has
Ship has
Ship haS
hcs Ship
HCS Ship
Hcs Ship
hCs Ship
hcS Ship
i hcs Ship
i HCS Ship
i Hcs Ship
i hCs Ship
i hcS Ship
Ship hcs
Ship HCS
Ship Hcs
Ship hCs
Ship hcS
hms Ship
HMS Ship
Hms Ship
hMs Ship
hmS Ship
i hms Ship
i HMS Ship
i Hms Ship
i hMs Ship
i hmS Ship
Ship hms
Ship HMS
Ship Hms
Ship hMs
Ship hmS
hms ms Ship
HMS MS Ship
Hms Ms Ship
hMs mS Ship
hmS ms Ship
i hms ms Ship
i HMS MS Ship
i Hms Ms Ship
i hMs mS Ship
i hmS ms Ship
Ship hms ms
Ship HMS MS
Ship Hms Ms
Ship hMs mS
Ship hmS ms
