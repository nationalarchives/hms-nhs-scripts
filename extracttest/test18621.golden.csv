data.text
""
Has Ship
Has Ship
Has Ship
Has Ship
Has Ship
I Has Ship
I Has Ship
I Has Ship
I Has Ship
I Has Ship
Ship Has
Ship Has
Ship Has
Ship Has
Ship Has
HCS Ship
HCS Ship
HCS Ship
HCS Ship
HCS Ship
I Hcs Ship
I Hcs Ship
I Hcs Ship
I Hcs Ship
I Hcs Ship
Ship Hcs
Ship Hcs
Ship Hcs
Ship Hcs
Ship Hcs
HMS Ship
HMS Ship
HMS Ship
HMS Ship
HMS Ship
I Hms Ship
I Hms Ship
I Hms Ship
I Hms Ship
I Hms Ship
Ship Hms
Ship Hms
Ship Hms
Ship Hms
Ship Hms
HMS Ship
HMS Ship
HMS Ship
HMS Ship
HMS Ship
I Hms Ms Ship
I Hms Ms Ship
I Hms Ms Ship
I Hms Ms Ship
I Hms Ms Ship
Ship Hms Ms
Ship Hms Ms
Ship Hms Ms
Ship Hms Ms
Ship Hms Ms
